src/BloomBrowserUI/bookEdit/readerSetup/readerSetup.ui.ts

```diff
@@ -490,6 +490,7 @@ function handleThingsToRemember(jqueryEvent: JQueryEventObject): void {
  */
 function updateSightWords(ta: HTMLInputElement): void {
   var words: string = ta.value.trim().replace(/ ( )+/g, ' '); // remove consecutive spaces
+  words = words.replace(/,/g, ''); // remove commas
```

Comment:
Do the four places we remove commas have something in common? I find it interesting, for example, that two of the four also remove multiple spaces. Could we find a place for a function something like CleanupSightWords()? Better still if you can make the method name indicate more clearly why they should be cleaned up.